mentors/typescript/c.json

```
@@ -5,4 +5,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "Using typescript professionally on projects, I love to network and help"
-  }]
+  },
+  {
+    "github_username": "ceddlyburge",
+    "name": "Cedd Burge",
+    "link_text": "Blog Posts",
+    "link_url": "https://medium.com/@cuddlyburger",
+    "avatar_url": https://github.com/ceddlyburge.png,
```

url need to be enclosed in double quotes `"`.
```suggestion
    "avatar_url": "https://github.com/ceddlyburge.png",
```